---
layout: page
title: ""
---

<!-- {% include JB/setup %} -->

Michael Love <img style="float: right;" src="http://mike-love.net/img/michaellove_sm.jpg"> <br>

michaelisaiahlove at gmail dot com <br>

Postdoctoral Fellow <br>
[Irizarry lab](http://rafalab.dfci.harvard.edu) <br>
Department of Biostatistics and Computational Biology, <br>
Dana-Farber Cancer Institute & <br>
Department of Biostatistics, <br>
Harvard TH Chan School of Public Health

[CV](http://mike-love.net/cv/love2015.pdf) <br>
[@mikelove](http://twitter.com/mikelove) <br>
[Google Scholar](https://scholar.google.com/citations?user=vzXv764AAAAJ)

---

### Research

I use statistical models to infer biologically meaningful
patterns from high-throughput sequencing data, and develop
open-source statistical software for the Bioconductor Project.
One of my main research efforts is in authoring and maintaining the
[DESeq2 package](http://bioconductor.org/packages/DESeq2) for statistical
analysis of RNA-seq experiments.

### Background

I completed a PhD in Computational Biology and Scientific Computing
(2013) in the Vingron Department at the Max Planck Institute for
Molecular Genetics in Berlin and the Mathematics and Informatics
Department of the Freie Universit√§t, Berlin.  I completed a Statistics
M.S. (2010) and Mathematics B.S. (2005) at Stanford University.

### Latest work

My latest work is studying the effect of fragment sequence bias
(e.g. fragment-level GC content bias) on
RNA-seq transcript abundance estimation. I have developed a novel
[method](http://biorxiv.org/content/early/2015/08/28/025767) and
[software](http://github.com/mikelove/alpine) for correcting fragment
sequence bias for the purposes of transcript quantification in
RNA-seq, which greatly reduces the technical artifacts and batch
effects when compared to state-of-the-art methods.
